### PR TITLE
Feat/get erc20 token

### PR DESCRIPTION
**Pull request by monitz87**
This PR:

- Renames the `DetailedERC20` exported class to `ERC20`
- Fixes some miswritten comments
- Exposes a `getERC20TokenWrapper` function from PolymathAPI that receives an address and returns an `ERC20` wrapper instance

### CODE REVIEW

File: src/PolymathAPI.ts [hunk 1]
```diff
@@ -206,6 +211,14 @@ export class PolymathAPI {
     });
   };
 
+  /**
+   * Get a wrapped token from an address
+   * @return TokenWrapper ERC20
+   */
+  public getERC20TokenWrapper = async (params: GetERC20WrapperParams): Promise<ERC20> => {
+    return this.tokenFactory.getERC20TokenInstanceFromAddress(params.address);
```
Comment (victorwiebe) [inline src/PolymathAPI.ts]: Shouldnt this be:
```
 public getERC20TokenWrapper = async (params: GetERC20WrapperParams): Promise<ERC20TokenWrapper> => {
```

Comment (monitz87) [reply src/PolymathAPI.ts]: Why? The class is imported as `ERC20` intentionally to match what is being exported by the index

Comment (victorwiebe) [reply src/PolymathAPI.ts]: Oh I see

**Pull request merged**